QV.nlogo

```diff
@@ -319,6 +329,43 @@ to assign-pmp-from-distribution
   )
 end
 
+to assign-correlated-pmp-from-distribution
+  ; beta distribution at low variance is similar to a normal distribution
+  ; assume pmp is normally distributed with mean = 0.5 and stdev = 0.1
+  ; 11/1/2020 adding majority? attribute for voters to determine z score of utility
+  ask voters [
+    let zscore 0
+    (ifelse
+      calibration = "manual" [
+        (ifelse majority? = true
+          [set zscore ((utility - majority-mean-utility) / majority-utility-stdev)]
+          [set zscore ((utility - minority-mean-utility) / minority-utility-stdev)]
+        )
+      ]
+      calibration = "1. prop8-mean>0" [
+        (ifelse majority? = true
+          [set zscore ((utility + 0.5) / 0.28)]
+          [set zscore ((utility - 0.865) / 4.5)]
+        )
+      ]
+      calibration = "2. prop8-mean=0" [
+        (ifelse majority? = true
+          [set zscore ((utility + 0.5) / 0.28)]
+          [set zscore ((utility - 0.77) / 5)]
+        )
+      ]
+    )
+    set zscore (abs zscore)
+
+    let uncorrelated-perceived-pivotality (random-normal 0.5 0.1)
+    (ifelse r > 0
```

Comment:
change `r` to `utility-pmp-correlation`. Also get rid of the correlate-utility-pmp? switch on the interface and just check of `utility-pmp-correlation = 0` on line 295.